RELEASE/scripts/autoscend/auto_king.ash

```diff
@@ -3,112 +3,77 @@ import <autoscend.ash>
-		foreach it in toPull
+	if(get_property("auto_borrowedTimeOnLiberation").to_boolean() && (get_property("_borrowedTimeUsed") == "false"))
+	{
+		if(get_property("_clipartSummons").to_int() < 3)
 		{
-			if(storage_amount(it) > 0)
-			{
-				auto_log_info("Pulling " + storage_amount(it) + " " + it, "green");
-			}
-			pullAll(it);
+			cli_execute("make borrowed time");
 		}
-
-		toPull = $items[Bittycar meatcar, burrowgrub hive, Can of Rain-Doh, Cheap Toaster, chester\'s bag of candy, chroner cross, chroner trigger, the cocktail shaker, Creepy Voodoo Doll, festive warbear bank, glass gnoll eye, infinite BACON machine, picky tweezers, taco dan\'s taco stand flier, Trivial Avocations Board Game, warbear breakfast machine, warbear soda machine];
-		foreach it in toPull
+		if((item_amount($item[Borrowed Time]) > 0) && (my_daycount() > 1))
 		{
-			if(storage_amount(it) > 0)
-			{
-				auto_log_info("Pulling/Using " + storage_amount(it) + " " + it, "green");
-			}
-			pullAndUse(it, 1);
+			use(1, $item[borrowed time]);
 		}
+	}
 
-		pullPVPJunk();
+	visit_url("campground.php?action=workshed");
 
-//		visit_url("lair2.php?preaction=key&whichkey=436");
-		visit_url("museum.php?action=icehouse", false);
-		visit_url("place.php?whichplace=desertbeach&action=db_nukehouse");
-		if((get_property("sidequestOrchardCompleted") != "none") && !get_property("_hippyMeatCollected").to_boolean())
+	if(get_property("auto_snapshot") == "")
+	{
+		if(svn_info("bumcheekascend-snapshot").last_changed_rev > 0)
 		{
-			visit_url("shop.php?whichshop=hippy");
+			cli_execute("snapshot");
 		}
```

Comment:
bumcheek snapshot is long dead, just ditch this.

Reply:
change made